bin/dev_setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Install components, setup DB and Solr:
+bundle install
+bundle exec rake argo:install
+
+# Run the server
+bundle exec rake jetty:stop  # Ensure it's not running
+bundle exec rake jetty:start # This may take a few minutes
+
+# Load and index records in the 'development' Solr core
+bundle exec rake argo:repo:load
```

Comment:
:+1: to that.

actually, what about calling it `argo:laptop:setup` and loading both the `development` and `test` repos/cores?  so basically the above, plus `RAILS_ENV=test bundle exec rake argo:repo:load`.  is there a way to invoke a rake task under a different env than the parent task without shelling out (since changing the env temporarily for the task invocation feels like a hack)?  i'd be fine with shelling out for the second load, since you wouldn't do that until after the regular repo load had run successfully, at which point you'd have already reaped the benefit of testing whether jetty was up.